src/Stratis.StratisD/Properties/launchSettings.json

```diff
@@ -1,7 +1,8 @@
 {
   "profiles": {
     "Stratis.StratisD": {
-      "commandName": "Project"
+      "commandName": "Project",
+      "commandLineArgs": "stratis -testnet"
```

Comment:
you don't need to specify `stratis` in stratisD and there's already a -tesntet launch settings.

Reply:
Fixed.